x-pack/plugin/ml/build.gradle

```diff
@@ -10,24 +12,40 @@ esplugin {
   extendedPlugins = ['x-pack-autoscaling', 'lang-painless']
 }
 
+def localRepo = providers.systemProperty('build.ml_cpp.repo').orNull
 
 repositories {
-  exclusiveContent {
-    forRepository {
```

Comment:
This was added in #70105 to avoid the risk of somebody uploading a malicious ml-cpp bundle to Maven Central or some other 3rd party repo we search, and then that getting downloaded in preference to what we've put in our own dedicated repo.  Is it possible to keep this functionality?

Reply:
Yes, I've added this back. It wasn't clear to me how this differed from `content.includeGroup` initially but reading the docs on this DSL cleared that up.